.github/blunderbuss.yml

```diff
@@ -25,4 +25,4 @@ assign_issues_by:
   - 'api: logging'
   - 'api: clouderrorreporting'
   to:
-  - googleapis/api-logging
+  - googleapis/api-logging-reviewers
```

Comment:
Feels a little odd that this is different than what is in CODEOWNERS, should that be updated too?